credit_card.rb

```diff
@@ -1,11 +1,39 @@
-# Creadit Card
+# Credit Card
 class CreditCard
-  attr_reader :balance
-  def initialize
+  attr_reader :balance, :deposit, :withdraw
+
+  def initialize(_balance, deposit, withdraw)
     @balance = 0.00
+    @deposit = deposit
+    @withdraw = withdraw
   end
 
-  def charge(amount)
+  def deposit(amount)
```

Comment:
The method we added during the assessment was `charge(amount)`.   Please undo this rename...